### PR TITLE
default fill for two color svg

### PR DESCRIPTION
**Pull request by hellatan**
if we have a multi-color svg, there should be default fills on the elements we are targeting in addition to classes so we can also affect the fill via css. 

without fill:
screen shot 2016-11-30 at 12 26 37 pm

with fill:
screen shot 2016-11-30 at 12 27 47 pm

also, this should be called `logo-mobile` to keep consistency with the current logo naming convention. Since that would be a breaking change, that will be updated in the next major rev.

cc @cormickjbrowne

### CODE REVIEW

**Comment by hellatan** [comment]
@nameofname updated stuff we discussed offline

**Comment by hellatan** [comment]
@nameofname oh wait, need to update a couple more svgs with classes

**Comment by hellatan** [comment]
@nameofname ok all set now

File: package.json [hunk 1]
```diff
@@ -34,10 +34,6 @@
     "url": "https://github.com/1stdibs/dibs-vg/issues"
   },
   "homepage": "https://github.com/1stdibs/dibs-vg#readme",
-  "dependencies": {
-    "backbone": "^1.3.3",
-    "jquery": "^2.2.3"
-  },
   "devDependencies": {
```
Comment (nameofname) [inline package.json]: @hellatan - it just occurred to me that we should probably have react as a peer dep. You agree?

**Comment by nameofname** [comment]
@hellatan weren't you going to change a file name too?

**Comment by hellatan** [comment]
peerDep - yes. file name change - no, that's going to be in the major rev

**Comment by hellatan** [comment]
@nameofname - peerDep is actually already there: 
```json
  "peerDependencies": {
    "react": "^15.0.2",
    "react-dom": "^15.0.2"
  }
```

**Comment by hellatan** [comment]
so this is good to merge whenever

**Comment by hellatan** [comment]
> + dibs-vg@1.22.1

**Pull request merged**